srcpkgs/ocs-url/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'ocs-url'
+pkgname=ocs-url
+version=3.1.0
+revision=7
+wrksrc="ocs-url-release-3.1.0"
+hostmakedepends="qt5-qmake qt5-svg-devel qt5-svg kdeclarative kdeclarative-devel"
+depends="qt5"
```

Comment:
`depends` should be detected automatically by the libraries it's linked against. Most of the `-devel` dependencies should go in `makedepends`.